contrib/ci/test.sh

```diff
@@ -77,7 +77,9 @@ docker exec $PG_JOB is_running
 puts-step "checking if minio has 5 backups"
 BACKUPS="$(docker exec $MINIO_JOB ls /home/minio/dbwal/basebackups_005/ | grep json)"
 NUM_BACKUPS="$(docker exec $MINIO_JOB ls /home/minio/dbwal/basebackups_005/ | grep -c json)"
-if [[ ! "$NUM_BACKUPS" -eq "5" ]]; then
+# NOTE (bacongobbler): the BACKUP_FREQUENCY is only 1 second, so we could technically be checking
+# in the middle of a backup. Instead of failing, let's consider N+1 backups an acceptable case
+if [[ ! "$NUM_BACKUPS" -eq "5" -a ! "$NUM_BACKUPS" -eq "6" ]]; then
```

Comment:
makes sense. Sounds like I'm mixing up /bin/sh and /bin/bash if statements again. Thanks!